shared/styles/style-guide.desktop.js

```diff
@@ -52,7 +51,10 @@ export const globalColorsDZ2 = {
   lightGrey2: '#f6f6f6',
 
   white: '#ffffff',
-  black: '#000000'
+  black: '#000000',
```

Comment:
@MarcoPolo You have black{75,60,40,20,10} defined but not white, and in the code you use an opacity transform instead of these variables -- so I guess we should either add the white opacities to the globalColors, or remove the black ones?